web/Views/Shared/_RegisterPagePartial.cshtml

```diff
@@ -1,93 +1,122 @@
+        <input asp-for="LastName" class="form-control"/>
+        <span asp-validation-for="LastName" class="text-danger"></span>
+    </div>
+</div>
+
+<div class="registerTab" id="Landlord details">
+    <div class="form-group my-3">
+        <label>Landlord type:</label>
+        @Html.DropDownList("LandlordType", new List<SelectListItem>
+        {
+            new() { Text = "Charity", Value = "Charity", Selected = true },
+            new() { Text = "Private Residential Sector", Value = "Private Residential Sector" },
+            new() { Text = "Registered Provider", Value = "Registered Provider" },
+            new() { Text = "Local Authority", Value = "Local Authority" }
+        })
+    </div>
+    <div class="form-group my-3">
+        <label>For/ Not for profit:</label>
+        @Html.DropDownList("IsLandlordForProfit", new List<SelectListItem>
+        {
+            new() { Text = "For Profit", Value = "true", Selected = true },
+            new() { Text = "Not For Profit", Value = "false" }
+        })
+    </div>
+    <div class="form-group my-3">
+        <label class="control-label">Company (optional)</label>
+        <input asp-for="CompanyName" class="form-control empty"/>
+        <span asp-validation-for="CompanyName" class="text-danger"></span>
+    </div>
+</div>
+
+<div class="registerTab" id="Contact details">
+    <div class="form-group my-3">
+        <label asp-for="Email" class="control-label"></label>
+        <input type="email" asp-for="Email" class="form-control"/>
+        <span asp-validation-for="Email" class="text-danger"></span>
+    </div>
+    <div class="form-group my-3">
+        <label asp-for="Phone" class="control-label"></label>
+        <input type="tel" asp-for="Phone" class="form-control"/>
+        <span asp-validation-for="Phone" class="text-danger"></span>
+    </div>
+</div>
+
+<div class="registerTab" id="ChangeAhead status" >
+   
+    @if (Model.LandlordProvidedCharterStatus)
+    {
+        <style>
+            div.hidetrue {}
+            div.hidefalse {display: none;}
+        </style>
+    }
+    else
+    {
+        <style>
+            div.hidetrue {display: none;}
+            div.hidefalse {}
+        </style>
+    }
+    
+    <p>ChangeAhead Charter</p>
+
+    <div class="my-3">
+        <label asp-for="LandlordProvidedCharterStatus" class="form-label">What is your ChangeAhead Charter status?</label>
+        <select asp-for="LandlordProvidedCharterStatus" class="form-select" required
+                id="Select" onclick="checkIfSelectedValueIsTarget()">
+            <option selected disabled value="">Charter Status</option>
+            <option value="true">Already approved</option>
+            <option value="false">Not yet approved</option>
+        </select>
+        <span asp-validation-for="LandlordProvidedCharterStatus" class="text-danger"></span>
+    </div>
+
+    <div id="ifTrue" class="form-group my-3 hidetrue">

```

Comment:
you could do something like `style="@(Model.LandlordProvidedCharterStatus ? "" : "display: none;")"`, I think that's a bit nicer than having the `<style>` block above